pyproject.toml

```diff
@@ -154,33 +154,24 @@ test = [
 
 # Development (./CONTRIBUTING.md)
 dev = [
-    "aiohttp-wsgi",
-    "boto3",
+    "aiohttp-debugtoolbar",
+    "diffoscope",
```

Comment:
This should be a differ dependency, not dev specific

Reply:
Moved!